api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -247,15 +248,23 @@ private void validateStringLength(String field, String fieldName, int max, int m
     }
   }
 
-  private void validateProfileFields(Profile profile) {
+  private void validateAndCleanProfile(Profile profile) throws BadRequestException {
+    profile.setDemographicSurvey(
+        Optional.ofNullable(profile.getDemographicSurvey()).orElse(new DemographicSurvey()));
+    profile.setInstitutionalAffiliations(
+        Optional.ofNullable(profile.getInstitutionalAffiliations()).orElse(new ArrayList()));
+
+    String userName = profile.getUsername();
+    if (userName == null || userName.length() < 3 || userName.length() > 64) {
```

Comment:
Let's make this check first.

Reply:
I'm not 100% sure what you meant, but I reshuffled things so the "validation" steps and the "cleanup" steps are more clearly delineated.

Reply:
thanks - was just a nit, no worries